examples/eth.rs

```diff
@@ -0,0 +1,65 @@
+use esp_idf_sys::{self as _}; // If using the `binstart` feature of `esp-idf-sys`, always keep this module imported
```

Comment:
Not necessary - remove.

Reply:
Hm. How about `use esp_idf_svc::sys::{self as _};`? If same story, I'll need to see what is going on...